README.md

```diff
@@ -68,6 +68,7 @@ postgresql_databases:
     uuid_ossp: yes      # flag to install the uuid-ossp extension on this database (yes/no)
     citext: yes         # flag to install the citext extension on this database (yes/no)
     encoding: "UTF-8"   # override global {{ postgresql_encoding }} variable per database
+    state: present      # optional; one of 'present', 'absent', 'dump', 'restore'
```

Comment:
If this is a string, then we should double-quote it.  In general booleans are unquoted, but all strings should be quoted.